rust/src/crypto.rs

```diff
@@ -111,4 +114,58 @@ impl Drop for PrivateKey {
             dht_privatekey_delete(&mut *self)
         }
     }
+}
+
+impl DhtCertificate {
+    pub fn import(file: &str) -> io::Result<Box<DhtCertificate>> {
```

Comment:
would be nice to be able to load from memory

Reply:
Done